eva_submission/nextflow/variant_load.nf

```diff
@@ -119,3 +120,69 @@ process load_vcf {
     java -Xmx4G -jar $params.jar.eva_pipeline --spring.config.location=file:$params.eva_pipeline_props --parameters.path=$variant_load_properties
     """
 }
+
+
+vcf_files_list = Channel.fromPath(params.valid_vcfs)
+            .splitCsv(header:true)
+            .map{row -> tuple(file(row.vcf_file), row.db_name)}
+
+
+/*
+ * Create properties files for Accession Import Job.
+ */
+process create_properties_for_acc_import_job {
+    input:
+    tuple vcf_file, db_name from vcf_files_list
+
+    output:
+    path "${acc_import_property_file_name}" into accession_import_props
+
+    exec:
+    props = new Properties()
+    params.acc_import_job_props.each { k, v ->
+        props.setProperty(k, v.toString())
+    }
+
+    accessioned_report_name = vcf_file.getFileName().toString().replace('.vcf','.accessioned.vcf')
```

Comment:
If I'm not mistaken only the compressed vcf gets [copied](https://github.com/EBIvariation/eva-submission/blob/e933cdfaff103458f565d95ae53254355a5a2be6/eva_submission/nextflow/accession.nf#L167) to the output directory in the automation.

Reply:
as discussed offline, seems fine